pom.xml

```diff
@@ -208,7 +208,7 @@
     <vers.hdp.solr>5.2.1</vers.hdp.solr>
     <solr.hdp.expected.version>5.2.1</solr.hdp.expected.version> <!-- sanity check to verify we actually run against the expected version rather than some outdated version -->
     <vers.hdp.tika>1.7</vers.hdp.tika>
-    <vers.hdp.jackson>2.3.1</vers.hdp.jackson>
+    <vers.hdp.jackson>2.7.4</vers.hdp.jackson>
```

Comment:
Are these version changes needed? Rocana uses jackson 1.9.13, so this doesn't seem to be for syncing with that repo